src/main/java/sirius/biz/jobs/batch/BatchProcessJobFactory.java

```diff
@@ -169,10 +170,20 @@ protected String getPenaltyToken() {
      * @throws Exception in case of any error which should abort this job
      */
     protected void executeTask(ProcessContext process) throws Exception {
+        logParameters(process);
         try (BatchJob job = createJob(process)) {
             job.execute();
         }
     }
 
     protected abstract BatchJob createJob(ProcessContext process) throws Exception;
+
+    private void logParameters(ProcessContext process) {
+        String output = "Parameter \"%s\": %s";
```

Comment:
use a stringbuilder and output everything a once like 
Parameters:
x: y
a: b
...